libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1269,6 +1270,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
      * @see [ViewUtils.capture] to capture screenshots
      * @see [FeedbackHelper.encodeScreenshot] to encode screenshots
      */
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
```

Comment:
Is this a breaking change? 🤔 

BTW we may consider it a bug fix refs. https://github.com/mapbox/mapbox-navigation-android/pull/5904#discussion_r889273632